src/main/java/wolox/training/TrainingApplication.java

```diff
@@ -2,7 +2,9 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
+@EnableJpaRepositories("com.baeldung.persistence.repo")
```

Comment:
I don't think the EnableJpaRepositories annotation is necessary

Reply:
okay